external-authority-sources.md

```diff
@@ -0,0 +1,275 @@
+# External sources Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Main Endpoint
+**/api/integration/externalsources**   
+
+Provide access to the configured external sources. It returns the list of existent external sources.
+
+Example:
+```json
+{
+  "_embedded": {
+    "externalsources": [
+      {
+        "id": "orcid",
+        "name": "orcid",
+        "hierarchical": false,
+        "type": "externalsource",
```

Comment:
I think a "metadata" attribute should be available here. In case you need to describe in more detail each source.

Reply:
From how I understand this based on the chat, this would be a single property. This may perhaps be better named "description" to reserve the term "metadata" for something where you have field names and values (e.g. https://github.com/DSpace/Rest7Contract/pull/74/files#diff-1f0dda62a7e6e5d84bb011254146955bR110)

It seems like a limited amount of work to have a configuration property for each source to describe it. Alternatively, this could be considered a UI aspect where internationalization labels are used to add detail per source (which would also ensure it can be displayed in the user's preferred language)

Reply:
> It seems like a limited amount of work to have a configuration property for each source to describe it. Alternatively, this could be considered a UI aspect where internationalization labels are used to add detail per source (which would also ensure it can be displayed in the user's preferred language)

But, considering we are supplying this source through REST API, that "description" should be implemented in all REST clients. But I'm ok with having it on the angular side.